### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by testacc992** [3kh0/ext-remover#480] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
On Extension Launcher (Bookmarklet) It doesn't download extensions it says "blocked by your admin"  but this hack only works on one extension I tried to download and it worked so I know it works but not on others Pls help -Roka.

![image](https://user-images.githubusercontent.com/94496154/216437612-31e9bf00-8ffd-4053-9a96-24ea0ae9761a.png)

**Comment by Aka-but-nice** [comment, marked as answer]
Yes, you have to not have a blocklist for this to work.

**Reply by testacc992** [reply]
what?

**Reply by 3kh0** [reply]
@testacc992 That is because your school has set a block list, try a different extension (imagine some stupid custom cursors mod) like ublock origin

**Reply by testacc992** [reply]
@3kh0 is there a way to see the blocked list? -roka

**Reply by ehunt34** [reply]
chrome://policy, under the URLBlocklist policy.
